ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/electra/statetransition/epoch/EpochProcessorElectra.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Consensys Software Inc., 2024
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.spec.logic.versions.electra.statetransition.epoch;
+
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.spec.config.SpecConfigBellatrix;
+import tech.pegasys.teku.spec.logic.common.helpers.BeaconStateMutators;
+import tech.pegasys.teku.spec.logic.common.statetransition.epoch.status.ValidatorStatus;
+import tech.pegasys.teku.spec.logic.common.statetransition.epoch.status.ValidatorStatusFactory;
+import tech.pegasys.teku.spec.logic.common.util.BeaconStateUtil;
+import tech.pegasys.teku.spec.logic.common.util.ValidatorsUtil;
+import tech.pegasys.teku.spec.logic.versions.altair.helpers.BeaconStateAccessorsAltair;
+import tech.pegasys.teku.spec.logic.versions.altair.helpers.MiscHelpersAltair;
+import tech.pegasys.teku.spec.logic.versions.bellatrix.statetransition.epoch.EpochProcessorBellatrix;
+import tech.pegasys.teku.spec.schemas.SchemaDefinitions;
+
+public class EpochProcessorElectra extends EpochProcessorBellatrix {
+
+  private final UInt64 minActivationBalance;
+
+  public EpochProcessorElectra(
+      SpecConfigBellatrix specConfig,
+      MiscHelpersAltair miscHelpers,
+      BeaconStateAccessorsAltair beaconStateAccessors,
+      BeaconStateMutators beaconStateMutators,
+      ValidatorsUtil validatorsUtil,
+      BeaconStateUtil beaconStateUtil,
+      ValidatorStatusFactory validatorStatusFactory,
+      SchemaDefinitions schemaDefinitions) {
```

Comment:
missing finals

Reply:
aag i really need to fix that generate macro